### DISCUSSION TITLE
Get Max value in table column

### DISCUSSION THREAD

**Discussion by tedr56** [Budibase/budibase#12080] [Help] [answered]
Hi all.
I have a counter for sets of 10 records in a budibase table.
The counter increments every new set of 10.
I'm trying to get the highest last value of this field to determine the value of the new set.
Any idea how to accomplish that?
I'm just getting used to bindings and variables but struggling with handlebars syntax.
Thanks in advance for any help.

**Comment by ConorWebb96** [comment, marked as answer]
Hey @tedr56,

You can use Javascript if that is easier for you. I'd need a bit more information in this case, as I don't really understand what you are trying to achieve.

- What components are you using?
- You mentioned a field, so is this 10 sets of forms that you are generating?
- What datasource are you using?
- What version of Budibase are you using?
- Are you self-host or cloud?

Most of the data sources support paginating their records. If you are using one of the data sources that have this functionality you can already increment records without the need to code anything.

<img width="314" alt="Screenshot 2023-10-17 at 08 37 13" src="https://github.com/Budibase/budibase/assets/126772285/316ef749-c659-41f6-82d6-1db2515727b9">
<img width="1347" alt="Screenshot 2023-10-17 at 08 38 23" src="https://github.com/Budibase/budibase/assets/126772285/b19fb5b6-3d81-4491-994e-7af022f4ec30">

I've supplied some screenshots below to show this in action. Hopefully this helps.

**Comment by tedr56** [comment]
Thank you for the answer. I think I'm not starting the right way.
